assembl/static2/css/components/posts.scss

```diff
@@ -215,15 +234,36 @@ $deleted-post-border: 1px solid $light-grey !default;
   }
 
   .post-left {
-    width: 93%;
-    padding: 40px;
+    width: 100%;
+    padding: 20px 10px 20px 30px;
+
+    .meta {
+      display: flex;
+
+      .author .assembl-icon-back-arrow {
+        font-size: 12px;
+        padding-left: 6px;
+        padding-right: 6px;
+      }
+
+      .published-date {
+        font-size: 12px;
+        font-weight: 300;
+        line-height: 1.91;
```

Comment:
see here you are not consistent in the line height definition, above you use PX here you use EM... make up your mind son :)